test/v0.2.0/LiquidityWallet.ts

```diff
@@ -0,0 +1,150 @@
+import { ethers, upgrades } from "hardhat";
+import { expect, use } from "chai";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { Contract } from 'ethers';
+
+describe("Grindery Liquidity Wallet", () => {
```

Comment:
The Smart Contract is not fully covered.

From your @tcoratger review I think will be a few changes to make it work in the way you expect. 
After your Review I can set new unit tests with the following changes.

Reply:
Done below